torch/_inductor/decomposition.py

```diff
@@ -44,15 +44,18 @@
         aten.hardsigmoid,
         aten.hardsigmoid_backward,
         aten.hardswish,
+        aten.hardswish_,
```

Comment:
cc @bdhirsh. The way these decompositions work in primTorch is they call fn with `out=self` kwarg, and, after going through a bunch of stuff it ends with
```
return copy_to.copy_(copy_from)
```
https://github.com/pytorch/pytorch/blob/b0cda0b38cf5e0516269e38314b1290fc1c26779/torch/_prims_common/wrappers.py#L173
so we can't use this as decomposition reintroduces inplace ops? (but the same thing was happening in the decompositions deleted here, so maybe that's ok?)

Reply:
AFAIK the problematic case is when a functionalized graph is decomposed and the decomposition re-introduces mutations. This shouldn't matter here since we are decomposing an operation that already mutates, so should never appear in a functionalized graph.

Reply:
Right, but then should we have decomps for inplace ops at all, or should we functionalize everything?

Reply:
I have similar concern as @ngimel - Should we even have the decomps for inplace op at Inductor level?